exec/jdbc-all/pom.xml

```diff
@@ -511,7 +511,7 @@
                   This is likely due to you adding new dependencies to a java-exec and not updating the excludes in this module. This is important as it minimizes the size of the dependency of Drill application users.
 
                   </message>
-                  <maxsize>39000000</maxsize>
+                  <maxsize>39100000</maxsize>
```

Comment:
Please rebase onto the master and revert this change, since there was PR, were excluded some classes from the JAR.